.github/workflows/development-ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Development to Development-CI Sync
+
+on:
```

Comment:
Could you also add `workflow_dispatch` here?

Reply:
sure